src/containers/AdminCampaignEdit/sections/CampaignInteractionStepsForm/index.tsx

```diff
@@ -389,9 +389,22 @@ const CampaignInteractionStepsForm: React.FC<InnerProps> = (props) => {
         title="What do you want to discuss?"
         subtitle="You can add scripts and questions and your texters can indicate responses from your contacts. For example, you might want to collect RSVPs to an event or find out whether to follow up about a different volunteer activity."
       />
-      <Box m={2}>
-        <ScriptPreviewButton campaignId={campaignId} />
-      </Box>
+      <div style={{ display: "flex", justifyContent: "space-between" }}>
```

Comment:
I looked through the onboarding and contributing docs but didn't see any guidance around css/styling, so I kept this as an inline change for simplicity's sake. I'm happy to change it if there's a convention I should be following, lmk!

Reply:
Yeah agree on no real convention. I've followed a rule of thumb of if it goes over one line - pull it out of the body. Curious if @hiemanshu has thoughts around this too.

Reply:
Suggestion: See if you can use a [`Grid`](https://v4.mui.com/components/grid/#grid) with Grid items to get the same outcome. [See example here](https://github.com/politics-rewired/Spoke/blob/main/src/containers/AdminCampaignStats/components/DeliverabilityStats.tsx)  Inline CSS for the div with `Box` containers for the buttons feels mismatched I think.